src/flowcean/core/report.py

```diff
@@ -135,6 +137,68 @@ def pretty_print(
             )
             console.print(panel)
 
+    def select_best_model(
+        self,
+        metric_name: str,
+        *,
+        mode: str = "maximize",
```

Comment:
This should use the `Literal` type-hint. To show what options for `mode` are allowed